pySNOM/images.py

```diff
@@ -142,29 +145,67 @@ def transform(self, data):
         else:
             return data - norm
 
+class RotatePhase(Transformation):
 
-class Process:
```

Comment:
This is cool. Just do not subclass Transormation, because your transform here uses different parameters. Just to make it more consistent. No need to subclass anything in this one.

Reply:
Or maybe I can unify this with the other ones. Also might naming might be misleading. The `phasedata` is here what we call data in the other transform methods. Maybe `amplitudedata` should be an attribute just like `degree` and then the transform becomes the same as for all the subclasses of Transform.

Reply:
You decide. It is fine to have an exception for this one if you deem both data sets equivalent.